census_view.cpp

```diff
@@ -1102,6 +1958,29 @@ wxWindow* CensusView::CreateChildWindow()
     return list_window_;
 }
 
+wxWindow* CensusGridView::CreateChildWindow()
+{
+    grid_window_ = new(wx) wxGrid(GetFrame(), wxID_ANY);
+    grid_window_->HideRowLabels();
+    grid_window_->SetUseNativeColLabels();
+
+    grid_window_->SetTable(grid_table_, true, wxGrid::wxGridSelectRows);
+
+    for(auto const& it : table_type_converter::get_all())
+        {
+        it.second->register_data_type(grid_window_);
+        }
+
+    grid_table_->SetAttrProvider(new wxGridCellAttrProvider());
```

Comment:
Couldn't these 2 lines be folded into `CensusGridTable` ctor?

If there is something preventing us from doing it there, there should be a comment saying something like "We can do this now that the grid is initialized" or whatever the real reason is.